abci/example/kvstore/kvstore.go

```diff
@@ -88,13 +79,28 @@ func NewPersistentApplication(dbDir string) *Application {
 	if err != nil {
 		panic(fmt.Errorf("failed to create persistent app at %s: %w", dbDir, err))
 	}
-	return NewApplication(db)
+	return NewApplication(db, DefaultLanes())
 }
 
 // NewInMemoryApplication creates a new application from an in memory database.
 // Nothing will be persisted.
 func NewInMemoryApplication() *Application {
```

Comment:
You need a new persistent application with and w/o lanes

Reply:
Done.